omas/tests/test_omas_physics.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# # -*- coding: utf-8 -*-
+
+"""
+Test script for omas/omas_physics.py
+python -m unittest test_omas_physics (from omas/omas/tests)
+python -m unittest discover omas (from omas top level; runs all tests)
+"""
+
+# Basic imports
+from __future__ import print_function, division, unicode_literals
+import os
+import unittest
+import numpy
+import warnings
+import copy
+
+# Plot imports
+import matplotlib as mpl
+from matplotlib import pyplot as plt
+
+# OMAS imports
+from omas import *
+from omas.omas_utils import *
+from omas.omas_physics import *
+
+
+class TestOmasPhysics(unittest.TestCase):
+    """
+    Test suite for omas_physics.py
+    """
+
+    # Flags to edit while testing
+    verbose = False  # Spammy, but occasionally useful for debugging a weird problem
+
+    # Sample data for use in tests
+    ods = ODS()
+
+    # Utilities for this test
+    def printv(self, *arg):
+        """Utility for tests to use"""
+        if self.verbose:
+            print(*arg)
+
+    def setUp(self):
+        test_id = self.id()
+        test_name = '.'.join(test_id.split('.')[-2:])
+        self.printv('{}...'.format(test_name))
+
+    def tearDown(self):
+        test_name = '.'.join(self.id().split('.')[-2:])
+        self.printv('    {} done.'.format(test_name))
+
+    @unittest.expectedFailure  # TODO: fix core_profiles_pressures or revise this test
+    def test_core_profiles_pressures(self):
+        ods2 = copy.deepcopy(self.ods)
+        ods2.sample_profiles(include_pressure=False)
+        ods3 = copy.deepcopy(self.ods)
+        updated_ods = core_profiles_pressures(ods2, update=False)
```

Comment:
@orso82 Calling core_profiles_pressures fails, so I flagged this test as broken. I'm not sure what the best way to fix this one is.